src/assets/style/module/changelog/_changelog.less

```diff
@@ -0,0 +1,97 @@
+.changelog-container {
+    width: 480px;
```

Comment:
please alphabetize these rules

Reply:
I started to standardize the dialog styling in https://github.com/openscope/openscope/pull/1192/files#diff-8b0e11a201d389745a7546e04a8dfedc

It would be awesome if you could add it to this PR so we don't have to refactor it later.

Reply:
For the record, @n8rzz  created a ticket for modal styling #1323